core/Access/Role/Admin.php

```diff
@@ -29,13 +29,13 @@ public function getId(): string
 
     public function getDescription(): string
     {
-        return Piwik::translate('UsersManager_PrivAdminDescription', array(
+        return Piwik::translate('UsersManager_PrivAdminDescription', [
             Piwik::translate('UsersManager_PrivWrite')
-        ));
+        ]);
     }
 
     public function getHelpUrl(): string
     {
-        return Url::addCampaignParametersToMatomoLink('https://matomo.org/faq/general/faq_69/');
+        return Url::addCampaignParametersToMatomoLink('https://matomo.org/faq/general/faq_69/') ?? '';
```

Comment:
That one does not make sense. The only way for `addCampaignParametersToMatomoLink` to return a `null` would be passing a null.

We could set the URL function return to a conditional type:

```
@return ($url is string ? string : null)
```

That would take care of every place the function is used.

Or with generics, but that feels like a bit too much:

```
@template T of string|null
@param T $url
@return T
```